tap_trello/streams.py

```diff
@@ -291,11 +309,22 @@ class Actions(DateWindowPaginated, ChildStream):
     key_properties = ["id"]
     replication_method = "INCREMENTAL"
     parent_class = Boards
+    MAX_API_RESPONSE_SIZE = 1000
+
+class Cards(ChildStream):
+    stream_id = "cards"
+    stream_name = "cards"
+    endpoint = "/boards/{}/cards/all"
+    key_properties = ["id"]
+    replication_method = "FULL_TABLE"
+    parent_class = Boards
+    MAX_API_RESPONSE_SIZE = 20000
```

Comment:
Looks like this isn't being used, the `_paginate_window` request needs updated to pass the limit in.

Reply:
We aren't using `_paginate_window`, `Cards` get synced by `Stream.get_records()`

Reply:
Maybe you meant to put this on the `Actions` class? So we looked there and and fixed it.

Reply:
Ah, yup, I meant actions